deploy/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       serviceAccountName: istio-operator
       containers:
         - name: istio-operator
-          image: gcr.io/istio-testing/operator:master-latest-daily
+          image: docker.io/sdake/operator:sdake
```

Comment:
note to change this to your own docker hub if testing manually - or alternately revert this part of the PR when the work is done.